pkgs/top-level/python-packages.nix

```diff
@@ -7742,6 +7810,25 @@ in {
     };
   };
 
+  requests-oauthlib = buildPythonPackage rec {
+    name = "${pname}-${version}";
```

Comment:
The `name` attribute is added by `buildPython*` and should therefore be removed.